zndraw/server/events.py

```diff
@@ -163,38 +173,40 @@ class Scene(BaseModel):
 
     return schema
 
-
 @io.on("modifier:run")
 def modifier_run(data):
     # emit entered the queue
-    running_jobs = app.config["MODIFIER"].get("num_jobs", 0) + 1
-    app.config["MODIFIER"]["num_jobs"] = running_jobs
+    JOB_ID = uuid4()
+    app.config["MODIFIER"]["queue"].append(JOB_ID)
+
     name = data["params"]["method"]["discriminator"]
     emit(
         "message:log",
-        f"Modifier {name} enqueued. {running_jobs} running jobs.",
+        f"Modifier {name} enqueued. {_get_queue_position(JOB_ID)} running jobs.",
         to=request.sid,
     )
```

Comment:
Do we want to keep that?

Reply:
Let's not duplicate information, feel free to remove